translations/Marathi/special-terms.md

```diff
@@ -0,0 +1,42 @@
+Group PM : ग्रुप खाजगी संदेश  , Transliteration used for group , <group name> मध्ये एक ग्रुप खाजगी संदेश पाठवा
+Realm : संघटन , used translation of organization for same reason as of chinese translation , एक संघटन बनवा
+Stream : स्ट्रीम  , used transliteration for better understanding , स्ट्रीममध्ये प्रवेश करा
+Topic : विषय , best word to signify the usage in zulip , नवीन विषय सुरू करा
+Private/Invite-Only Stream : खाजगी / केवळ-आमंत्रित स्ट्रीम , used translation of "private/invited-only" for better understanding , आपले खाजगी / केवळ-आमंत्रित स्ट्रीम
+Public Stream : सार्वजनिक स्ट्रीम  , best translation of public for usage in zulip , आपले सार्वजनिक स्ट्रीम 
+Bot : बॉट , used transliteration , स्वागत बॉट
+Integration : अनुप्रयोग एकीकरण , used translation of "application integration" , फेसबुक सह झुलीप अनुप्रयोग एकात्मता
+Notification : सूचना , translates to information but is understandable by user , सूचना सक्रिय करा
+Alert Word : सतर्क शब्द , only translation which is understandable by average user , आपले सतर्क शब्द
+View : पहा , used translation of see for implication that it is verb , संदेश पहा
+Home : मुख्यपृष्ठ , used translation of homepage instead for better understanding , मुख्यपृष्ठावर जा
+Emoji : इमोजी , used transliteration as there is no translation of this word , इमोजी शैली
+
+Phrases
+
+Subscribe to a stream/Unsubscribe from a stream : एखाद्या स्ट्रीमची सदस्यता घ्या / एक स्ट्रीम मधुन सदस्यता रद्द करा , only translation available , <stream name> हया स्ट्रीमची सदस्यता घ्या 
+Narrow to : याद्वारे फिल्टर करा , used translation of filter by , याद्वारे फिल्टर करा
+Filter : फिल्टर , used transliteration , याद्वारे फिल्टर करा
+Mute/Unmute : सूचना चालू / बंद करा , used translation of "turn on/turn off notifications", सूचना चालू / बंद करा
+Deactivate/Reactivate : बंद करा / सुरू करा  ,  used translation of "Turn OFF / ON", आपले खाते बंद करा / सुरू करा
+Search : शोध , same as translation used in google search , वापरकर्त्यासाठी शोधा
+Pin : पिन करा , the second word force-implies the verb form , आपले पोस्ट पिन करा
+Mention/@mention : उल्लेख / @ उल्लेख , best translation of word , आपले उल्लेख
+Invalid : चूक , Used translation of incorrect , हे नाव चुकीचे आहे
+Customization : पसंती , translation shown in translate.google.com(if different from this) is in possessive form , आपले पसंती
+I want : मला <something> पाहिजे , Something SHOULD come between the two words , मला तुम्हाला एक मेल पाठवायचा आहे
+User : वापरकर्ता , only translation which is short and straight-forward (others mean the one who uses, etc) , वापरकर्त्यासाठी शोधा
+Person/People : व्यक्ती / लोक , note that there is no similarity between the different forms as there is no other plural/singular equivalent , लोकांमध्ये शोधा
+
+Other
+
+You : आपण/तूम्ही , both forms are generally acepted, however note that only तू  is used for younger people/anyone we generally don't respect as elders , तुम्हाला भेटून आनंद झाला
```

Comment:
Here, it should be 'तु' in  'तूम्ही' instead of 'तू'

Reply:
Thanks for the heads up